front_end/html/edit_exercise.html

```diff
@@ -262,6 +403,24 @@ <h2>Create exercise</h2>
             </div>
         </div>
 
+        <div class="shadow-box">
+            <div>
+                <p><strong>Pre-execution logic:</strong></p>
```

Comment:
(this block is part of the instructor_checking_code pull request but I had to leave it in so it didn't throw errors)